### DISCUSSION TITLE
Data/Grid View doesn't show all rows because the vertical scrollbar isn't fully visible when the Status widget is below it

### DISCUSSION THREAD

**Discussion by yurivict** [pawelsalawa/sqlitestudio#4355] [Q&A]
When the Status widget is present it appears on top of the Data/Grid View obscuring its portion.
It is impossible to scroll to the bottom because the Status widget covers the bottom.

The usual behavior is that the scroll view should be always fully visible.

**Comment by pawelsalawa** [comment]
Is it possible, that you have the internal windows normalized (not maximized)?

**Comment by yurivict** [comment]
It is just hanging over the GridView.

**Comment by pawelsalawa** [comment]
Can you show screenshot of the entire SQLiteStudio window?

**Comment by yurivict** [comment]
![image](https://user-images.githubusercontent.com/271906/147708794-f06064ee-c7f6-4d99-9985-e9f43b5a53ea.png)

**Comment by yurivict** [comment]
Scroll bar isn't visible because it is under the Status widget.

**Comment by pawelsalawa** [comment]
Your status field is "flowing", not docked. Grab status field by its title bar and drag it to the bottom of the whole SQLiteStudio window to dock it at the bottom.

**Comment by yurivict** [comment]
I know that this is a flowing Qt5 widget.
But for many regular users this situation would easily become a puzzle.

Maybe flowing widgets should not be allowed to be aligned with other widgets to make them look more prominent, not sure.

**Reply by pawelsalawa** [reply]
If widget is flowing, it's not possible to limit it on where it can or cannot be placed.
If you try hard you can get a lot of functionalities to be misleading, and not only in SQLiteStudio.
I don't think this case is common, as it was reported only first time and this is possible in SQLiteStudio for around 7 years.
